### PR TITLE
Adds readme + build steps for api spec

### PR DESCRIPTION
**Pull request by carsonfarmer**
This is just an experimental idea. Let me know what you think @jsign ? If you think its a good idea, we might want to actually create a Makefile and do this (and the SQL spec) build stuff properly.

### CODE REVIEW

**Comment by jsign** [comment]
@carsonfarmer , Nice, very useful. I agree with the Makefile rule.

Two things:
- I'll tombstone https://github.com/orgs/tablelandnetwork/projects/12 (looks like it's the same as the below bullet)
- I think this is targeting https://github.com/tablelandnetwork/docs/issues/8 ?

File: README.md [hunk 1]
```diff
@@ -47,6 +47,27 @@ pandoc -s --toc \
...
+The OpenAPI specification document is used to generate clients and services that drive the Tabeland Validator. It can also be used to generate Markdown documentation via:
+
+```shell
+npm i -g widdershins
+cd specs/validator
+widdershins tableland-openapi-spec.yaml \
+    --omitHeader \
+    --httpsnippet \
+    --language_tabs \
+    'javascript:JavaScript:fetch' 'javascript--node:Node.JS' 'go:Go' 'shell:cURL:curl' \
```
Comment (jsign) [inline README.md:63]: This is just a comment, but maybe it's good only to include `shell:cURL;curl` since we have nicer SDKs for JS and Go?

**Review by jsign — APPROVED** [review]

File: README.md (hunk 1 above)
Comment (carsonfarmer) [reply README.md]: yeh good call. it is also kinda messy with that many options.

**Comment by carsonfarmer** [comment]
Cool, I'm going to merge this just because it'll be nice to have these available. I'll create an issue to do the Makefile setup later (Makefiles are outside my zone of expertise!).

**Pull request merged**